torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -82,8 +82,12 @@ Reducer::Reducer(
            variable_index++) {
         auto& variable = replicas_[replica_index][variable_index];
         const auto index = VariableIndex{
+#ifdef _WIN32
+            replica_index, variable_index
```

Comment:
is this because MSVC does not support the following format?

Reply:
yes, exactly.